src/util/chunked_content_decoder.js

```diff
@@ -82,6 +134,22 @@ class ChunkedContentDecoder extends stream.Transform {
                 } else {
                     this.state = STATE_READ_CHUNK_HEADER;
                 }
+            } else if (this.state === STATE_READ_TRAILER_DATA) {
+                const start_trailer = index;
+                for (; index < buf.length; ++index) {
+                    if (buf[index] === CR_CODE) {
+                        this.state = STATE_WAIT_NL_TRAILER;
+                        break;
+                    }
+                    this.chunk_trailer_str = buf.toString('utf8', start_trailer, index);
+                }
+            } else if (this.state === STATE_WAIT_NL_TRAILER) {
+                if (buf[index] !== NL_CODE) return this.error_state();
+                this.trailers.push(this.chunk_trailer_str);
```

Comment:
also need to set chunk_trailer_str to '' here.

Reply:
I would write like this
```suggestion
                if (this.chunk_trailer_str) {
                    this.trailers.push(this.chunk_trailer_str);
                    this.chunk_trailer_str = '';
                }
```